README.md

```diff
@@ -32,6 +32,44 @@ using the simple python tool from the `save` directory. The loader is a simple
 shell script so that you don't need python installed when loading the
 dashboards.
 
+
+Create a new dashboard
+----------------------
+
+If you added support for a new protocol in Packetbeat or a module in
+Metricbeat, it would be nice to create a dedicated Kibana dashboard to 
+prove the functionality of your data.
+The Kibana dashboards are saved in a special index in Elasticsearch. By default
+it's .kibana, but it can be configured to anything else.
+
+The first step, in creating your own Kibana dashboard, would be to load 
+all the current dashboards in Kibana using the
+load.sh script. Then, create the Kibana dashboard using Kibana functionalities
```

Comment:
This part is bit hard to follow. The phrase "using Kibana functionalities" seems a little vague, and users probably aren't familiar with the kibana_dump.py script (I'm not). I understand that I'm supposed to use Kibana to create a new dashboard, but I need a few more details. For example, why do I need to load the other dashboards first? Is it mainly so that the index patterns get created? Am I expected to somehow build visualizations for my new dashboard by looking at the sample dashboards?